pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "devops-course-starter"
+version = "0.1.0"
+description = ""
+authors = ["JoeWFisher <joseph.fisher@softwire.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+pytest = "^6.0.1"
+datetime = "^4.3"
+python-dateutil = "^2.8.1"
+flask = "^1.1.2"
+requests = "^2.24.0"
+python-dotenv = "^0.12.0"
+
+[tool.poetry.dev-dependencies]
+
```

Comment:
If you include your test dependencies here, you could actually move your project to poetry completely - delete requirements.txt, setup.sh (and the gitignored env folder if you have that)

Rather than using 2 package managers simultaneously.